src/edu/csus/ecs/pc2/ui/TeamStatusPane.java

```diff
@@ -321,25 +326,65 @@ private void repopulateGrid(boolean populate) {
             teamLabel.setToolTipText(toolTipText);
             centerPane.add(teamName, teamLabel);
         }
+        
+        
+        /**
+         * Update counts for each status
+         */
+        
+        submittedClarsLabel.setText("Submitted Clar(s) (" + toInt( teamStatusCount.get(TeamStatus.TEAM_HAS_SUBMITTED_CLARS_ONLY), 0) + ")");
+        submittedRunsLabel.setText("Submitted Run(s) (" +toInt(  teamStatusCount.get(TeamStatus.TEAM_HAS_SUBMITTED_RUNS_ONLY) , 0) + ")");
+        hasLoggedInLabel.setText("Has Logged In (" + toInt( teamStatusCount.get(TeamStatus.TEAM_HAS_LOGGED_IN), 0 ) + ")");
+        nocontactLabel.setText("No Contact (" + toInt( teamStatusCount.get(TeamStatus.NO_TEAM_CONTACT), 0) + ")");
+        readyLabel.setText("READY (" + toInt( teamStatusCount.get(TeamStatus.TEAM_HAS_SUBMITTED_RUNS_AND_CLARS), -0) + ")");
```

Comment:
The value "-0" (negative zero) for the default is confusing...

Reply:
Looks like a typo.   

In the future I think you should state that this type of mistake should be fixed to avoid
this sort of comment being mistaken for a remark rather than an indication of correction.

Reply:
pushed fix.